src/main/java/org/takes/rq/RqLive.java

```diff
@@ -113,4 +114,29 @@ public InputStream body() {
         };
     }
 
+    /**
+     * Check line and add header o replace last one.
+     * @param head All previous headers
+     * @param header Current header
+     * @throws HttpException if multiline do not start with space
+     */
+    private static void checkAndAddHeader(final List<String> head,
+            final String header) throws HttpException {
+        if (head.isEmpty() || header.contains(":")) {
+            head.add(header);
+        } else if (header.charAt(0) == ' ' || header.charAt(0) == '\t') {
+            final String prev = head.remove(head.size() - 1);
+            head.add(prev + header);
```

Comment:
@lautarobock Let's [avoid string concatenation[(http://www.yegor256.com/2014/06/19/avoid-string-concatenation.html). Also, we can join the previous line with this, avoiding the single use variable.